specs/.eslintrc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../.eslintrc.json",
+  "env": {
+    "jasmine": true
+  },
+  "rules": {
+    "no-unused-vars": "off"
```

Comment:
Yes, there are many cases in this repository where variables are really unused and should be removed

In most places where there is a false positive, disable the rule at the line level, except for files like `addDefaultMatchers` where is makes more sense at the file level.

Thanks.